src/keystore/InMemoryKeystore.ts

```diff
@@ -579,4 +593,44 @@ export default class InMemoryKeystore implements Keystore {
   lookupTopic(topic: string) {
     return this.v2Store.lookup(topic)
   }
+
+  async getV2ConversationHmacKeys(): Promise<keystore.GetConversationHmacKeysResponse> {
```

Comment:
Will this be exposed to the app developer, and if so, would it make sense to allow them to supply a list of topics to fetch keys for?

Reply:
yes, this will be exposed. and yes, that seems like a good option to have.

Reply:
added here: [fix: allow specific topics when getting HMAC keys](https://github.com/xmtp/xmtp-js/pull/519/commits/5c59492dcbe5c4bf6bbeb9320551690e53cff3f8)

Reply:
Great improvement